### ISSUE TITLE
Upping the font size of the page for readability causes the test to fail

### ISSUE THREAD

**Issue by wjodon** [FreeCodeCamp/FreeCodeCamp#6027]
Challenge [Waypoint: Change the Font Size of an Element](http://www.freecodecamp.com/challenges/waypoint-change-the-font-size-of-an-element) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/601.3.9 (KHTML, like Gecko) Version/9.0.2 Safari/537.86.3</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

For those of us with "elder vision" the size of the default font is uncomfortably small.  Increasing the size of the font on the page for readability also affects the font size of the result content in the paragraphs and thus causes the test to the challenge results to fail.  Interestingly, the font sizes are slightly smaller rather than larger as would be expected.

![screenshot-failedtest](https://cloud.githubusercontent.com/assets/89663/12223389/2ae8b968-b7a4-11e5-8ad9-1318774ede71.png)
![screenshot-failedtestparagraphelementcomputedproperties](https://cloud.githubusercontent.com/assets/89663/12223390/2aebcd4c-b7a4-11e5-9494-6b151e0d8b7b.png)
![screenshot-succeededtest](https://cloud.githubusercontent.com/assets/89663/12223391/2aec7c42-b7a4-11e5-9ff9-4044ff067206.png)
![screenshot-succeededtestparagraphelementcomputedproperties](https://cloud.githubusercontent.com/assets/89663/12223388/2ae87ebc-b7a4-11e5-815d-cfedf04c25dc.png)

My code:
```html
<style>
  .red-text {
    color: red;
  }
  p {
    font-size: 16px;
  }
</style>

<h2 class="red-text">CatPhotoApp</h2>

<p class="red-text">Kitty ipsum dolor sit amet, shed everywhere shed everywhere stretching attack your ankles chase the red dot, hairball run catnip eat the grass sniff.</p>

<p>Purr jump eat the grass rip the couch scratched sunbathe, shed everywhere rip the couch sleep in the sink fluffy fur catnip scratched.</p>
```

**Comment by SaintPeter** [comment]
I was unable to reproduce this issue with your code.  I tested on both Chrome and Firefox.  How is it that you are changing the font size?  I used Ctrl-+ on both browsers to increase the zoom.

**Comment by ltegman** [comment]
I can repro with View > Zoom > Zoom Font Only and then zooming in.

We've had this issue on another CSS waypoint and the solution was to change the test to accept values within 2px + or - of the 'correct' answer. Looks like doing that with this one as well would solve that problem.

**Comment by SaintPeter** [comment]
Is that in Firefox only?  I couldn't find the "Zoom Font Only" in Chrome.

I'm not sure how fixable this is going to be, though, because the issue with the +/- X pixels was probably ok for a border, but for a font size, how much variation are we going to accept?

**Comment by ltegman** [comment]
@SaintPeter I can only see it in Firefox. I don't see an issue with +-2. The specific number they type in for font size is unimportant to the learning outcome. Unless our starting font size for the phone is already 14px?

**Comment by SaintPeter** [comment]
Ok, played with this a bit.  I think we can just say "16px or greater".  The default is 14px.

**Comment by wjodon** [comment]
Safari using command-+

**Comment by wjodon** [comment]
Should have told you - it was in Safari using command-+ to change the font.
